base/tomcat-10.1/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -0,0 +1,8 @@
+eclipse.preferences.version=1
```

Comment:
This file is not needed. There's already an Eclipse project in the top-level folder.